llvm/test/CodeGen/AArch64/ext-narrow-index.ll

```diff
@@ -1,5 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc < %s -mtriple=aarch64 | FileCheck %s
+; RUN: llc < %s -mtriple=aarch64 | FileCheck %s --check-prefix=CHECK
+; RUN: llc < %s -global-isel -mtriple=aarch64 | FileCheck %s --check-prefix=CHECK-GISEL
```

Comment:
This can use  `--check-prefixes=CHECK,CHECK-SD` for the first line and ` --check-prefixes=CHECK,CHECK-GI` for the second. Where they are identical the check lines should then fold to both using `CHECK`